tests/functional/adapter/test_model_grants.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from dbt.context.base import BaseContext  # diff_of_two_dicts only
+from dbt.tests.adapter.grants.test_model_grants import BaseModelGrants
+
+
+@pytest.mark.hive
+# TODO: setup roles in Galaxy
```

Comment:
Is this a TODO relevant for this PR ?

Reply:
This is a reminder to test grants with SEP and Galaxy.

Reply:
Please include issue number in the comment here.

Reply:
Added issues for both Galaxy and SEP tests.